app/src/main/AndroidManifest.xml

```diff
@@ -51,12 +51,24 @@
         </activity-alias>
 
         <!--
-          Fennec declared entry for homescreen pinned shortcuts.
+          Fennec's activity set to handle http/s schemes.
+          Need to also use it in Fenix to keep the previously default app status after migration.
+          Also the Fennec declared entry for homescreen pinned shortcuts.
         -->
         <activity-alias
             android:name="org.mozilla.gecko.BrowserApp"
             android:targetActivity=".IntentReceiverActivity"
             android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <category android:name="mozilla.components.pwa.category.SHORTCUT" />
```

Comment:
Doesn't that break PWAs that need to go through `IntentReceiverActivity`? (Haven't tested the patch yet)

Reply:
Right. Thanks. Just tested and indeed the PWAs are lost.